spec/phi_attrs/phi_record/class__phi_allowed_spec.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+RSpec.describe 'class phi_allowed?' do
```

Comment:
Looks like lass phi_allowed? exists now? You should update the documentation in this PR then.

Reply:
Ah, no, that's not added. I recommend renaming the description to `phi_allowed? works with class allow_phi methods`

Reply:
I rearranged this to make it better. The class one is now empty, and everything was merged into the instance file with two context.